### PR TITLE
Add  `communicating-with-the-host` tutorial

### PR DESCRIPTION
**Pull request by tcaiazzi**
Add a tutorial to explain how to connect a network scenario to the host through the Kathará external feature. 

The procedure can be used to connect a network scenario directly to a host interface or to a VM, a container, or a process running on the host. 

This feature can be used to develop complex scenarios involving also VM, or containers outside the Kathará emulation.

### CODE REVIEW

File: tutorials/kathara-external/communicating-with-the-host/README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# Kathará External - Communicating with the host (Linux Only)
+
+This tutorial shows a simple example on how to allow a network scenario to directly communicate with a host interface.
+
+The same procedure can be used to connect a network scenario to a VM, a container, or another network scenario running
+on the host executing the emulation.
+
+The tutorial works only on Linux. It works even on WSL, if you install Docker directly from Linux
```
Comment (Skazza94) [inline tutorials/kathara-external/communicating-with-the-host/README.md:8]: I would rephrase in: 
"The tutorial only works on Linux. On WSL, the tutorial works only if you install Docker directly in the Linux distribution (Docker Desktop on the Windows host is not supported)."

File: tutorials/kathara-external/communicating-with-the-host/virtual-topology.png
Comment (Skazza94) [inline tutorials/kathara-external/communicating-with-the-host/virtual-topology.png:1]: The image is a screenshot of PowerPoint (the grid lines are visible). Can you save the slide as PNG and upload it?

**Review by Skazza94 — CHANGES_REQUESTED** [review]

File: tutorials/kathara-external/communicating-with-the-host/README.md (hunk 1 above)
Comment (tcaiazzi) [reply tutorials/kathara-external/communicating-with-the-host/README.md]: Updated

File: tutorials/kathara-external/communicating-with-the-host/virtual-topology.png
Comment (tcaiazzi) [reply tutorials/kathara-external/communicating-with-the-host/virtual-topology.png:1]: Uploaded

**Review by Skazza94 — APPROVED** [review]

**Review by lorenzo93 — APPROVED** [review]
